Assignment2A/ArithmeticOperation.java

```diff
@@ -0,0 +1,47 @@
+package Assignment2A;
+
+import java.io.IOException;
+
+/**
+ * This Interface provide generic method for arithmetic operation  
+ *
+ */
+public interface ArithmeticOperation {
+	
+	/**
+	 * 

```

Comment:
Please try to add purpose of method in comment and follow the order of comment like purpose then param then return detail of method in sequence.

Reply:
ok sir